src/ServicePulse.Host/vue/src/components/failedmessages/MessageView.vue

```diff
@@ -62,25 +65,26 @@ function getConfiguration() {
     })
     .then((data) => {
       configuration.value = data;
-      return;
+      return getEditAndRetryConfig();
```

Comment:
Can this be done in parallel to the first data load?

E.g. can we make both requests at the same time?


Something like:

```JavaScript
function getConfiguration() {
  return promise.all([useFetchFromServiceControl("configuration")
    .then((response) => response.json())
    .then((data) => {
      configuration.value = data;
    }), getEditAndRetryConfig()]);
```

Reply:
We could do it this way, but I would definitely have to update some other things.  In the `getEditAndRetryConfig()` I set the edit and retry configuration to `configuration.value.edit`, but if the regular configuration returns after the edit and retry config, which sets the root object of the configuration (`configuration.value`), then the edit and retry config will be lost.  I could set the regular configuration to something like `configuration.value.message` and the edit and retry config to `configuration.value.edit`, which should make it so that it wouldn't matter what order the requests finish.